dlp/snippets/deid/deid_test.go

```diff
@@ -477,3 +477,27 @@ func getUnwrappedKey(t *testing.T) (string, error) {
 	return string(encodedKey), nil
 
 }
+
+func TestReidTableDataWithFPE(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	buf := new(bytes.Buffer)
+
+	keyRingName, err := createKeyRing(t, tc.ProjectID)
+	if err != nil {
+		t.Fatal(err)
+	}
+	keyFileName, cryptoKeyName, keyVersion, err := createKey(t, tc.ProjectID, keyRingName)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer destroyKey(t, tc.ProjectID, keyVersion)
+
+	if err := reidTableDataWithFPE(buf, tc.ProjectID, keyFileName, cryptoKeyName); err != nil {
+		t.Fatal(err)
+	}
+	got := buf.String()
+	if want := "Table after re-identification "; !strings.Contains(got, want) {
+		t.Errorf("TestReidTableDataWithFPE got %q, want %q", got, want)
```

Comment:
We can also confirm that "90511" is not present in the output. The re-identified value should be different.

Reply:
Updated the test case according to that.